packages/components/src/atoms/Price/Price.tsx

```diff
@@ -53,7 +53,7 @@ export const Price = forwardRef<Omit<HTMLSpanElement, 'children'>, PriceProps>(
         ref={ref}
         data-fs-price
         data-testid={testId}
-        data-variant={variant}
+        data-fs-variant={variant}
```

Comment:
```suggestion
        data-fs-price-variant={variant}
```


WTYT to make it more specific?